changelog/change_improve_naming_method_name_example.md

```diff
@@ -0,0 +1 @@
+* [#11678](https://github.com/rubocop/rubocop/pull/11678): Improve `Naming/MethodName` example `AllowedPatterns`. ([@sambostock][])
```

Comment:
Change log entry is not required because no user impact. Can you remove it?

Reply:
Done!